portfolio/src/main/webapp/index.html

```diff
@@ -25,7 +25,12 @@ <h1>Tal's Portfolio</h1>
       <div id="fact-container"></div>
  
       <p>Post a comment here!</p>
-      <form method="POST" action="/data">
+      <div id = 'notLoggedIn'>
+          <p>Login To Comment</p>
+          <button target="/user">Login</button>
```

Comment:
Are there any params you need to send to the server to login here?  I think the `loginUrl` from your java line 48 is what you might need to link to here.

Reply:
Yes loginURL is what I want them to be redirected to if they are not logged in. When you say "send it to server" does that mean I replace target='/user' with target='loginURL'?

Reply:
> replace target='/user' with target='loginURL'?

*yes*, but you don't want 'loginURL' (with quotes) which would just be the word 'loginURL'.  What you want is the URL that was stored in loginURL coming from your `fetch()`.  You'd need to use JS to pull the loginURL value from the response, then update your target here.